tools/benchmark/main.py

```diff
@@ -127,8 +127,8 @@ def set_throughput_streams():
                 if args.number_threads and is_flag_set_in_command_line("nthreads"):
                     config[device]['CPU_THREADS_NUM'] = str(args.number_threads)
 
-                if is_flag_set_in_command_line("enforcebf16") or is_flag_set_in_command_line("enforce_bfloat16"):
-                    config[device]['ENFORCE_BF16'] = 'YES' if args.enforce_bfloat16 else 'NO'
+                if (is_flag_set_in_command_line("enforcebf16") or is_flag_set_in_command_line("enforce_bfloat16")) and args.enforce_bfloat16 == False:
```

Comment:
Please rollback this change as well. User should have possibility set YES via command line.  The same about python. You need modify only help message.

Reply:
A key behavior of key ENFORCE_BF16 was changed by default. Now the key unavailable for non CPX and enabled by default on CPX. So, we need just an ability to disable BF16

Reply:
Currently benchmark_app has 2 ways to set parameters - via configuration json file and directly via command line. 

Command line arguments has the first priority and should set config parameter to YES or NO depends on value. You allows only NO setting. Your modification brakes logic of application. Why did you write in help that 2 values are possible (True and False) if only one value is used?

Configuration file has the second priority. By default benchmark_app doesn't set this parameter at all. So any default value can be used in IE or not used at all, it is doesn't matter from application point of view. 

I have validated application and all scenarios work well without code modification. We only need to specify how user can set value in help.

Reply:
Anyway of you would like to have string flag instead of boolean (small not backward compatible change) it is better use YES and NO values directly. As it is done for pin flag, for example.

Reply:
Reverted